CMakeLists.txt

```diff
@@ -35,21 +41,21 @@ target_include_directories(
   lyra
   INTERFACE
   $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
-  $<INSTALL_INTERFACE:include>
+  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
```

Comment:
`CMAKE_INSTALL_INCLUDEDIR` will make installation of this library non-portable, since it will hardcode the absolute include path in the Config file. I do not like this.

Reply:
Not absolute, but relative to the prefix. For absolute ones `_FULL_` vars are used.